server/app/views/v1/host_nodes/token.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.id @node.node_id
```

Comment:
Using the `node_id` here feels particularly broken, because in the most common use-cases of this API, it's simply going to be `null`.... :face_with_head_bandage: